apps/maestro/src/features/AddErc20/hooks/useDeployAndRegisterRemoteStandardizedTokenMutation.ts

```diff
@@ -102,14 +98,15 @@ export function useDeployAndRegisterRemoteStandardizedTokenMutation(
       return [deployTxData];
     }
 
-    const registerTxData = input.destinationChainIds.map((chainId, i) =>
-      encodeInterchainTokenServiceDeployAndRegisterRemoteStandardizedTokenData({
-        ...baseArgs,
-        distributor: "0x", // remote tokens cannot be minted, so the distributor must be 0x
-        operator: deployer,
-        destinationChain: chainId,
-        gasValue: input.gasFees[i] ?? BigInt(0),
-      })
+    const registerTxData = input.destinationChainIds.map(
+      (destinationChain, i) =>
+        INTERCHAIN_TOKEN_FACTORY_ENCODERS.deployRemoteInterchainToken.data({
+          destinationChain: destinationChain,
+          gasValue: input.gasFees[i] ?? BigInt(0),
+          distributor: deployer,
```

Comment:
we should propmpt the user whether to allow the remote tokens to be mintable by the an address (default to deployer)